models/orders.sql

```diff
@@ -0,0 +1,38 @@
+    with orders as (
+        select * from {{ ref('stg_orders') }}
+    ),
+
+    payments as (
+        select * from {{ ref('stg_payments') }}
+    ),
+
+    order_payments as (
+        select
+            order_id,
+            sum(amount) as amount
+        from payments
+        group by 1
+    ),
+
+    final as(
+
+        select 
+            orders.order_id,
+            orders.customer_id,
+            orders.order_date,
+            orders.status,
+            coalesce(order_payments.amount, 0 ) as amount
+        from orders
+        left join order_payments using (order_id)

```

Comment:
I know they mentioned that this isn't the ideal way to do it and instead compare the two order_ids, but I didn't bother to change mine either. Just wanted to point it out.